_posts/camazotz/2016-01-27-NavaneetsCh4Exercises.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+author: camazotz
+title: "Navaneet's Chapter 4 Exercises!"
+---
+
+Chapter 4 Exercise 6:
+
+<iframe src="https://trinket.io/embed/python/9d34362266" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Chapter 4 Exercise 7:
+
+<iframe src="https://trinket.io/embed/python/775e165270" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Ah looks like this got stuffed in with the turtle exercise.  See me if you're confused about how to make separate pull requests.  I'm assuming it was just an oversight.